### PR TITLE
Fix command in Source File Version Updater job

### PR DESCRIPTION
**Pull request by aggarw13**
The "Raise a Pull-Request" step of the Source File Version Updater GH workflow has an incomplete package installation command for the `gh` command line tool. This PR completes the command.

### CODE REVIEW

**Review by sarenameas — DISMISSED** [review]

File: .github/workflows/version-updater.yml [hunk 1]
```diff
@@ -34,7 +34,8 @@ jobs:
           git commit -am 'Update versioning in file from ${{ github.event.inputs.old_version_number }} to ${{ github.event.inputs.new_version_number }}'
           git push --set-upstream origin updater-job/update-to-${{ github.event.inputs.new_version_number }}
       - name: Raise a Pull-Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          sudo apt-get install
-          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+          sudo apt-get install gh
```
Comment (muneebahmed10) [inline .github/workflows/version-updater.yml:40]: Does this actually work on these instances? This isn't how `gh` is installed according to the official [documentation](https://github.com/cli/cli/blob/trunk/docs/install_linux.md), and running this on my vm results in "E: Unable to locate package gh"

Comment (sarenameas) [reply .github/workflows/version-updater.yml:40]:
```suggestion
          sudo apt-get install gitsome
```

Comment (aggarw13) [inline .github/workflows/version-updater.yml:40]:
```suggestion
          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key C99B11DEB97541F0
          sudo apt-add-repository https://cli.github.com/packages
          sudo apt update
          sudo apt-get install gh
```

**Review by muneebahmed10 — APPROVED** [review]

File: .github/workflows/version-updater.yml (hunk 1 above)
Comment (cobusve) [reply .github/workflows/version-updater.yml:43]: Does not look like this has been addressed?

Comment (aggarw13) [reply .github/workflows/version-updater.yml:43]: Made changes to follow official documentation.

**Review by cobusve — APPROVED** [review]

**Pull request merged**